From 272db68e131a20652c98453b75b9c743cda2b327 Mon Sep 17 00:00:00 2001
From: Rob Bradford <rob@linux.intel.com>
Date: Fri, 19 Sep 2014 17:06:00 +0100
Subject: [PATCH 3/3] xml-parser: Add missing break in switch statement

This would result in an inappropriate debug message appearing on valid
text data.
---
 rest/rest-xml-parser.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rest/rest-xml-parser.c b/rest/rest-xml-parser.c
index 49b49b4..4447a1a 100644
--- a/rest/rest-xml-parser.c
+++ b/rest/rest-xml-parser.c
@@ -197,6 +197,7 @@ rest_xml_parser_parse_from_data (RestXmlParser *parser,
         cur_node->content = g_strdup (G(xmlTextReaderConstValue (reader)));
         REST_DEBUG (XML_PARSER, "Text content found: %s",
                  cur_node->content);
+        break;
       default:
         REST_DEBUG (XML_PARSER, "Found unknown content with type: 0x%x", 
                  xmlTextReaderNodeType (reader));
-- 
2.3.1

